src/test/scala/projectEuler/LargestPrimeFactorSpec.scala

```diff
@@ -0,0 +1,21 @@
+package projectEuler
+
+import org.specs2.mutable.Specification
```

Comment:
Generally within the team we use (ScalaTest)[http://scalatest.org/]. I'd recommend switching to that.